### PR TITLE
T3W1 support

### PR DESCRIPTION
**Pull request by grdddj**
Adding support for the new emulator model.

NOTE: needs to have a `trezorlib` installed from firmware repo, as the released version does not support it yet.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -7,7 +7,7 @@ authors = ["SatoshiLabs <info@satoshilabs.com>"]
 [tool.poetry.dependencies]
 python = "^3.11"
 termcolor = "^1.1.0"
-trezor = "^0.13.9"
+trezor = { git = "https://github.com/trezor/trezor-firmware.git", branch = "main", subdirectory = "python" }
```
Comment (vdovhanych) [inline pyproject.toml:10]: Could we maybe do a beta release instead fo doing this?? It increases the size of the built image significantly.

Comment (grdddj) [reply pyproject.toml:10]: It is up to @matejcik to do the `trezorlib` release - I mentioned it to @Hannsek that it is/will be needed to do. I was checking the image size and now it seems to be exactly the same as with released version - could be that `poetry` improved the git-repo installation process

Comment (vdovhanych) [reply pyproject.toml:10]: Ok lets merge it

**Review by vdovhanych — APPROVED** [review]
lets merge it

**Pull request merged**